src/group_theory/subgroup/pointwise.lean

```diff
@@ -130,6 +131,27 @@ begin
   exact H.mul_mem hh hh',
 end
 
+lemma conj_act_normal {G : Type*} [group G] {H : subgroup G} (hH : H.normal ) (g : conj_act G) :
```

Comment:
```suggestion
lemma normal.conj_act {G : Type*} [group G] {H : subgroup G} (hH : H.normal ) (g : conj_act G) :
```
To have dot notation (not sure it works...).

Reply:
that works.